src/pages/admin/pages/faq/FaqPage.vue

```diff
@@ -3,16 +3,18 @@
     <va-card class="mb-8">
```

Comment:
I don't see this section implemented at all

<img width="426" alt="image" src="https://github.com/epicmaxco/vuestic-admin/assets/5394573/9fb8f7d8-c6c3-4f26-a33b-d5ed43296a65">

Reply:
![localhost_5173_pages_faq](https://github.com/epicmaxco/vuestic-admin/assets/29259987/bcbbc4ac-e4d6-44f7-ba6e-62e6dbc859be)
- made visual updates, maybe missed something but tried my best.
- there is still issue with icons(will check them tomorrow)

Reply:
For sidebar issue, we gonna need to use VaLayout component.

Reply:
@m0ksem if you speak about sidebar "height" issue it won't affect in browser experience, the sidebar doesn't move on scroll. I just changed(not in the code!) right column overflow value to make full page screenshot.